src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -141,6 +142,48 @@ const Content = (props: {
     );
   }
 
+  if (relevantGuidedStep.id === "ScanInProgress") {
```

Comment:
And so here you wouldn't make this conditional on the step (since it's not a different step in the guided experience flow), but on the actual prerequisite for showing this content:

```suggestion
  if (
    relevantGuidedStep.id === "ScanResult" &&
    props.stepDeterminationData.latestScanData?.scan?.onerep_scan_status ===
      "in_progress"
  ) {
```

Reply:
Got it — removed the step in [f0213a3](https://github.com/mozilla/blurts-server/pull/3431/commits/f0213a332d5120e489fc10fc112be1f9506ae62d).